lib/MusicBrainz/Server/Form/User/EditProfile.pm

```diff
@@ -7,9 +7,15 @@ use MusicBrainz::Server::Translation qw( l ln );
 use MusicBrainz::Server::Validation qw( is_valid_url );
 
 extends 'MusicBrainz::Server::Form';
+with 'MusicBrainz::Server::Form::Role::ValidateUsername';
 
 has '+name' => ( default => 'profile' );
 
+has_field 'username' => (
```

Comment:
I expect this will let everyone change their own username. (If not, how/where is it prevented?)